CRM/PivotReport/Entity.php

```diff
@@ -16,7 +18,25 @@ class CRM_PivotReport_Entity {
       ),
     ),
     'Membership' => array(),
-    'Contact' => array(),
+    'People' => array(
+      'extensions' => array(
+        'org.civicrm.hrjobcontract'
+      ),
+    ),
+    'Leave' => array(
+      'extensions' => array(
+        'uk.co.compucorp.civicrm.hrleaveandabsences',
+        'org.civicrm.hrjobcontract'
+      ),
+    ),
+//    'Sample' => array(
+//      'hookable' => [
```

Comment:
I don't understand how this make the extension hookable. How will other extensions be able to add information to this array and let the extension know that they also provide reports?

Reply:
I updated the PR by adding a hook to the Pivot extension callable by other extensions to add their own report entities.

Reply:
So, if I'm getting this right, with the new hook you've added, people will be able to add new definitions to this `$entities` array. Is that right?

Reply:
Yes, I tried adding the report files in the L&A extension and used the hook to inject the report definations and it worked fine.